Quickstart.md

```diff
@@ -18,6 +18,9 @@ If you are encoding from an image sequence (e.g. imagefile.0000.png imagefile.00
 name: test_quickstart
 sources: 
 - sourceimages/chip-chart-1080-noicc.png.yml
+comparisontest:
+   testtype: idiff
```

Comment:
What if this was a list rather than a dictionary, so that you could potentially even chain tests together and have a test harness at the end?

Separately, since each test returns different values, perhaps we could have a "validateresults" test that would allow you to define valid ranges of values:
```
comparisontest:
   - testtype: vmaf
   - testtype: idiff
      testtemplate: idiff -fail 0.00195 {originalfile} {newfile}
   - testtype: assertresults
      tests:
           - assert: between
              value: psnr_y
              between: [69, 71]  # psnr_y values between 69 and 71
           - assert: notzero
              value: file_size

Reply:
Daniel, take a look at the most recent commit, it should have this installed. Look at 
 EncodingGuidelines/enctests/test_configs/color_tests.yml as an example with the testing in place. I've stopped using the -fail flag, instead doing the "fail" in the assertresults test, that way we can see the values even if it passes.